src/app/icd/client/DefaultICDClientStorage.cpp

```diff
@@ -344,8 +350,18 @@ CHIP_ERROR DefaultICDClientStorage::SerializeToTlv(TLV::TLVWriter & writer, cons
     return writer.EndContainer(arrayType);
 }
 
+CHIP_ERROR DefaultICDClientStorage::CheckFabricExistence(FabricIndex fabricIndex)
```

Comment:
This should probaby return a boolean.  That will make the uses much less weirdly written.

Reply:
agree, done, thanks